scripts/db_stop.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Why not do it as part of makefile target?

Reply:
we can also do it inside makefile but doing it with script give us a clean code, good use of conditional statements and separate from makefile logic so that it would not effect our logic in case of any issue in removing the containers moreover we can use this script anytime.